src/utilities.ts

```diff
@@ -0,0 +1,5 @@
+import type { GeoJSONBaseGeometry, GeoJSONGeometry } from './types';
+
+export const flatGeometryCollection = (v: GeoJSONGeometry): GeoJSONBaseGeometry => {
```

Comment:
Is this function really needed in general?

Reply:
it can be useful for nested geometry collections
you can use `flatten` in `turf`, but it only goes up to one level depth (it crashes if it's more deep) and it returns a `feature collection` not a `geometry collection`